RubberduckTests/Refactoring/EncapsulateField/EncapsulateFieldConflictFinderTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using NUnit.Framework;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Refactorings;
+using Rubberduck.Refactorings.EncapsulateField;
+using Rubberduck.VBEditor.SafeComWrappers;
+using RubberduckTests.Mocks;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RubberduckTests.Refactoring.EncapsulateField
+{
+    [TestFixture]
+    public class EncapsulateFieldConflictFinderTests
+    {
+        private EncapsulateFieldTestSupport Support { get; } = new EncapsulateFieldTestSupport();
+
+        [SetUp]
+        public void ExecutesBeforeAllTests()
+        {
+            Support.ResetResolver();
```

Comment:
This seems to indicate that the `Support` variable has state although it is shared between tests. That is generally a bad idea because it means that the outcome of tests executed in parallel is basically undefined.

Reply:
PR name: I guess I am missing the other effects you are seeing. Can you say a bit more?

That said, your observation here represents a new issue and is definitely something to correct.  I had not considered that tests would/could be run in parallel.  As-is, I only ensure that no resolver state is propagated to tests run sequentially (in any order).  

Which makes me wonder when, if ever, using the `Setup/Teardown` facilities of `NUnit` would be appropriate.  I'm, thinking for RD they never are.  Regardless, I'd like to address this issue as a separate PR.